modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommercePriceLists.testcase

```diff
@@ -1093,14 +1093,272 @@ definition {
 			value1 = "7.45");
 	}
 
-	@description = "This is a test for COMMERCE-6220. When creating a price list only catalogs for which the user has permission should be visible in the drop down"
-	@ignore = "true"
+	@description = "This is a test for COMMERCE-8568. Verify multiple Price Entries can be set on the same Price List for the same SKU"
+	@priority = "3"
+	test AllowMultiplePriceEntriesInTheSamePriceList {
+		property osgi.module.configuration.file.names = "com.liferay.commerce.price.list.configuration.CommercePriceEntryConfiguration.config";
```

Comment:
```suggestion
		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false";
		property osgi.module.configuration.file.names = "com.liferay.commerce.price.list.configuration.CommercePriceEntryConfiguration.config";
```
`jsonws.web.service.paths.excludes=` is needed for `CommerceEntry.addAccountEntryUser`.
`company.security.strangers.verify=false` can be used instead of lines 1103-1110.